src/hooks/useCombobox/index.js

```diff
@@ -309,6 +309,7 @@ function useCombobox(userProps = {}) {
       [refKey]: handleRefs(ref, toggleButtonNode => {
         toggleButtonRef.current = toggleButtonNode
       }),
+      disabled,
```

Comment:
actually, for consistency, can we remove the `disabled` from the props spread, use `rest.disabled` for the check and then it will get passed automatically?